Orange/widgets/data/owselectrows.py

```diff
@@ -73,7 +73,7 @@ def __init__(self):
         self.cond_list = QtGui.QTableWidget(
             box, showGrid=False, selectionMode=QtGui.QTableWidget.NoSelection)
         box.layout().addWidget(self.cond_list)
-        self.cond_list.setColumnCount(3)
+        self.cond_list.setColumnCount(4)
```

Comment:
@RachitKansal, we're cleaning up the pull requests, so I wanted to ask whether you'll have time to continue working on this issue or can somebody else wrap it up?